core/domain/exp_domain.py

```diff
@@ -647,7 +650,11 @@ def create_default_exploration(
 
         # Calculate proto_size_in_bytes.
         proto_size_in_bytes = 0
-
+        Parse(json.dumps({
+            'name': 'Akshya',
+            'id': 1,
+            'email': 'gmail',
+        }), Person.Person())
```

Comment:
```py
        proto_message = Parse(json.dumps({
            'name': 'Akshya',
            'id': 1,
            'email': 'gmail',
        }), Person.Person())
       print(proto_message)
       print(proto_message.ByteSize())
```

Reply:
As discussed, this approach likely won't work since:

1. The current JSON representation doesn't match proto's JSON representation of structures
2. The structure itself for the proto representations aren't the same as the JSON structures, so they don't have an automatic direct mapping